angular/src/components/icon.component.ts

```diff
@@ -14,12 +14,12 @@ import { StateService } from 'jslib-common/abstractions/state.service';
 import { Utils } from 'jslib-common/misc/utils';
 
 const IconMap: any = {
-    'fa-globe': String.fromCharCode(0xf0ac),
-    'fa-sticky-note-o': String.fromCharCode(0xf24a),
-    'fa-id-card-o': String.fromCharCode(0xf2c3),
-    'fa-credit-card': String.fromCharCode(0xf09d),
-    'fa-android': String.fromCharCode(0xf17b),
-    'fa-apple': String.fromCharCode(0xf179),
+    'bwi-globe': String.fromCharCode(0xe909),
```

Comment:
Can we use a constant here for the icons?

Reply:
+1, missed that, thanks Oscar!

Reply:
Updated - thanks ya'll!